### PR TITLE
many: renaming related to inclusive language part 2

### PR DESCRIPTION
**Pull request by stolowski**
More fixes for inclusive language.

### CODE REVIEW

**Review by MiguelPires — APPROVED** [review]
LGTM, although woke seems to be failing

File: tests/main/snap-disconnect/test-snap-consumer.v2/meta/snap.yaml [hunk 1]
```diff
@@ -2,4 +2,4 @@ name: test-snap-consumer
 version: 2
 plugs:
     other:
-        interface: dummy
+        interface: empty
```
Comment (bboozzoo) [inline tests/main/snap-disconnect/test-snap-consumer.v2/meta/snap.yaml:5]: Just as a sanity check maybe it's useful to reach out to the store team to check if there aren't any snaps using that interface.

**Review by bboozzoo — APPROVED** [review]
Thanks

File: tests/main/snap-disconnect/test-snap-consumer.v2/meta/snap.yaml (hunk 1 above)
Comment (bboozzoo) [reply tests/main/snap-disconnect/test-snap-consumer.v2/meta/snap.yaml:5]: As discussed on MM, review-tools complain about dummy iface so snap upload would be impossible.

**Pull request merged**